sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -63,7 +63,17 @@ async def test_async_queue_by_queue_client_conn_str_receive_handler_peeklock(sel
             with pytest.raises(ServiceBusConnectionError):
                 await (sb_client.get_queue_receiver(servicebus_queue.name, session_id="test", max_wait_time=5))._open_with_retry()
 
+            with pytest.raises(ValueError):
```

Comment:
I'm debating whether having a comparable check for subscription receiver passes my bar for "worth being obnoxious about".  I'll leave that to your judgement and rubber stamp this regardless; if you decide to put it there as well feel free to ping me again for a quick OK.

Reply:
good callout, just added several lines of code in subscription receiver tests.